shotover-proxy/benches/mod.rs

```diff
@@ -0,0 +1,58 @@
+use cassandra_cpp::{stmt, Session};
+pub use test_helpers::docker_compose::DockerCompose;
+
+#[path = "../tests/helpers/mod.rs"]
+mod helpers;
+pub use helpers::ShotoverManager;
+
+pub struct BenchResources {
```

Comment:
I feel like combining redis and cassandra bench resources into a single struct just complicates things.
As is, despite having fields for both cassandra and redis connections the API does not allow both to be used at the same time.
I think we would be better off with:
```rust
// defined in redis_benches.rs
BenchResources {
    _compose: DockerCompose,
    _shotover_manager: ShotoverManager,
    pub connection: redis::Connection,
}

// defined in cassandra_benches.rs
BenchResources {
    _compose: DockerCompose,
    _shotover_manager: ShotoverManager,
    pub connection: Session,
}
```

Unless theres some advantage im missing?

Reply:
It was an attempt to de-duplicate the code given it's a very simple struct that performs the exact same function in both cases, enable the ability to extend it in cases where we want to benchmark both connections at the same time and add extra functionality to `BenchResources`.

However I don't feel strongly about this and perhaps it's over-engineering as you've pointed out so I'll revert it.